includes/payment-methods/class-wc-stripe-upe-payment-method.php

```diff
@@ -258,10 +258,13 @@ public function is_enabled_at_checkout( $order_id = null, $account_domestic_curr
 	/**
 	 * Returns the supported customer locations for which charges for a payment method can be processed.
 	 *
-	 * @return array
+	 * @return array Supported customer locations.
 	 */
-	public function get_countries() {
```

Comment:
This function was only used to populate the JS data with a list of eligible countries. Given I've changed it's purpose to return billing countries, I changed the name. Because this function is unreleased, there are no backwards compatibility concerns here.